downloads/swan-lake-release-notes/swan-lake-2201.9.0/RELEASE_NOTE.md

```diff
@@ -613,6 +613,38 @@ $= future<int> result = start name();
 
 #### CLI
 
+#### EDI tool
+    
+- Added support for EDIFACT to Ballerina schema conversion.
```

Comment:
Can be headings since we have lengthy explanations after?

Reply:
Yes. +1 from me. I changed it with [14889b3](https://github.com/ballerina-platform/ballerina-dev-website/pull/8938/commits/14889b3d14079f1f591ba3767a169ae10bdc2607). but seems like It doesn't align with other tool notes, isn't it?  What should we do?

Reply:
Hmm, looks like the OpenAPI changes are in point form, right? Shall we change back to that for consistency then?

Reply:
Done with [e1e1d19](https://github.com/ballerina-platform/ballerina-dev-website/pull/8938/commits/e1e1d1922818fbfd27016eaeb709ac6bd553b186).